dd-sdk-android/src/main/kotlin/com/datadog/android/rum/internal/monitor/DatadogRumMonitor.kt

```diff
@@ -52,26 +53,30 @@ internal class DatadogRumMonitor(
     // region RumMonitor
 
     override fun startView(key: Any, name: String, attributes: Map<String, Any?>) {
+        val eventTime = (attributes[TIMESTAMP] as? Long)?.asTime() ?: Time()
```

Comment:
Shouldn't we also remove the `TIMESTAMP` attribute in all `RumMonitor` APIs? Otherwise this attribute gets passed along with other attributes, then gets attached to the active view scope and gets overwritten there on every next resource, error or action. At the end it is send to server, and (?) gets displayed in the RUM Explorer.